dockers/docker-sonic-telemetry/base_image_files/monit_telemetry

```diff
@@ -2,4 +2,4 @@
 ## Monit configuration for telemetry container
 ###############################################################################
 check program container_memory_telemetry with path "/usr/bin/memory_checker telemetry 419430400"
-    if status == 3 for 10 times within 20 cycles then exec "/usr/bin/restart_service telemetry"
+    if status == 3 for 10 times within 20 cycles then exec "/usr/bin/restart_service telemetry" repeat every 2 cycles
```

Comment:
>2 [](http://example.com/codeflow?start=109&length=1)

How do you choose the magic number 2?

Reply:
`repeat every 2 cycles` at here means if Monit failed to reset its counter and memory usage of telemetry is still larger than the threshold (at here `400MB`) for `2 minutes`, then telemetry container will be restarted. 

I selected the number `2` since memory usage of telemetry possibly increased very quickly from MB to around GB within 2 minutes and telemetry should be restarted ASAP. 

Other numbers can be selected as well and do you have any suggestion?